libs/cdk/internals/scheduler/src/lib/scheduler.ts

```diff
@@ -1,6 +1,6 @@
 // see https://raw.githubusercontent.com/facebook/react/master/packages/scheduler/src/forks/SchedulerDOM.js
 
-import { ɵglobal } from '@angular/core';
+import { ɵɵdirectiveInject as inject, NgZone, ɵglobal } from '@angular/core';
```

Comment:
Minor: `ɵɵdirectiveInject as inject` is imported but unused.

Reply:
ty